### DISCUSSION TITLE
Share

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4730] [General] [answered]
Guys, tell me, is there a function to restrict or prohibit the ability to share links from the Wiki to some users? It would be great to include such rights as necessary for some employees.

**Comment by tommoor** [comment, marked as answer]
Users with only viewer permissions are not able to pulbicly share documents. It is also possible to restrict sharing for all team members at the collection level under `Permissions`.

**Comment by MaksGill** [comment]
Thanks for the answer, but I can't find the permissions section. Tell me, is it in the Wiki settings or the collection itself? It would be convenient to look at the screenshot.

**Reply by tommoor** [reply]
<img width="418" alt="image" src="https://user-images.githubusercontent.com/380914/213233447-a4996a46-7c4e-4a4d-8c36-faf82d4b7e62.png">

**Comment by MaksGill** [comment]
Thank you for the screenshot and for your answer. Restricted access to the collection to the view only level, assigned the user the observer role and then the participant role. In the Security section, I disabled Document Sharing. But all the same, users with all roles were able to share a link in a collection with a view-only access level. They couldn't edit it, yes. Can you tell me how this can be solved?

**Reply by tommoor** [reply]
The share button always appears, but if access to public sharing is disabled then it will just share an internal link which is similar to copying the url from the address bar, it does not give any extra permissions and does not work for logged out users.

**Comment by MaksGill** [comment]
Thank you for the answer!
